docs/source/howto/run_codes.rst

```diff
@@ -219,43 +219,83 @@ Doing so will prevent AiiDA from connecting to the given computer to check the s
 How to setup a code
```

Comment:
You may want to change this to
```suggestion
How to create a code
```
in order to be consistent with the cli.
The file name / anchors don't need to be changed (that way we don't need to update links).